### PR TITLE
Acceptance criteria steps

### PR DESCRIPTION
**Pull request by vlebedeff**

### CODE REVIEW

**Comment by vlebedeff** [comment]
@danshultz PTAL

**Comment by danschultz** [comment]
could you upload the compiled project to a github page? thanks

**Comment by vlebedeff** [comment]
@danschultz https://vlebedeff.github.io/ts-training

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "malevich",
+  "version": "1.0.0",
+  "description": "TypeScript / React / HTML5 Training Exercise",
+  "main": "./dist/bundle.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (danschultz) [inline package.json:8]: npm supports pre/post hooks. it'd be nice to use this to autoinstall the typings when you run `npm install`. see https://docs.npmjs.com/misc/scripts

File: src/redux/reducers.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  return (state: CanvasState = initialState, action: Action) => {
+    switch(action.type) {
+      case Actions.ADD_FIGURE:
+        return {
+          figures: [ ...state.figures, (action as Actions.AddFigureAction).figure ]
+        }
+
+      case Actions.REMOVE_FIGURE:
+        var index = (action as Actions.RemoveFigureAction).index;
+        var figures = state.figures;
```
Comment (danschultz) [inline src/redux/reducers.ts]: it's recommended to use `let` or `const` instead of `var`. these will be scoped to the block

File: package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "devDependencies": {
+    "css-loader": "^0.23.1",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "postcss": "^5.0.21",
+    "postcss-loader": "^0.9.1",
+    "postcss-nested": "^1.0.0",
+    "source-map-loader": "^0.1.5",
+    "style-loader": "^0.13.1",
+    "ts-loader": "^0.8.2",
```
Comment (danschultz) [inline package.json:36]: you're missing a dev dep on typescript

Comment (vlebedeff) [inline package.json:36]: I've linked it to the global installation. Will cover it

Comment (danschultz) [inline package.json:36]: better to use a local dep. i tried building your project and it failed to compile because typescript wasn't found.

**Comment by danschultz** [comment]
LGTM, just some minor comments

**Pull request merged**